ale/drivers/base.py

```diff
@@ -36,7 +34,8 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         this is done, the object is out of scope and the
         kernels can be unloaded.
         """
-        spice.unload(self.metakernel)
+        if self.metakernel:
```

Comment:
Should we warn in the instance where `self.metakernel` is None?

Reply:
In my PR, this is moved to the Spice specific mixin. As there is no case where you use the Spice mixin and not expect to load kernels, I imagine if no metakernel is found, the property should simply error.

Reply:
:+1: That sounds totally reasonable.